src/sqlBible/src/models/IBibleOutput.interface.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    BiblePhrase,
+    BibleSection,
+    IBibleReferenceRangeVersion,
+    IBibleReferenceRangeNormalized,
+    BibleBook,
+    BibleVersion,
+    IBibleReferenceVersion,
+    IBibleReferenceNormalized
+} from '.';
+
+export interface IBibleTextBase {
+    version: BibleVersion;
+    versionBook: BibleBook;
+    rangeVersion: IBibleReferenceRangeVersion;
+    rangeNormalized: IBibleReferenceRangeNormalized;
+    sectionLevel1?: BibleSection;
+    sectionLevel2Up?: BibleSection[];
```

Comment:
not super clear what this means

Reply:
`sectionLevel2Up`

Reply:
yea, not super proud of this. the thing i'm trying to solve here is how to deal with the dynamic number of sections-levels that a phrase can belong to. a bible can theoretically have a structure that is 4 or 5 levels deep. thinking of the UI, i think the most common use case is that we want to display level 1, that is the one that usually contains the heading (with level 0 being the paragraph level). the idea is to group all remaining levels in an array. hence the naming.